src/robocoup/commands/utilization.js

```diff
@@ -1,25 +1,185 @@
 /*
- * A command to show the utilization status of the company today.
+ * A command to show the utilization status of the company and yourself today.
  */
+import Promise from 'bluebird';
 import moment from 'moment';
+import R from 'ramda';
 import {createCommand} from 'chatter';
 import {query} from '../../lib/db';
-import heredoc from 'heredoc-tag';
+import {histogramByPercentage} from '../../lib/formatting';
+
+// filtering functions
+const isFuture = R.propEq('timeframe', 'future');
+const isPast = R.propEq('timeframe', 'past');
+const isBillable = R.propEq('is_billable', true);
+const isNonBillable = R.propEq('is_billable', false);
+const hasNoDays = R.pipe(R.prop('total'), R.equals(0));
+const hasDays = R.pipe(hasNoDays, R.not);
+const getTotal = R.pipe(R.map(R.prop('total')), R.sum);
+
+// date ranges
+const firstOfYear = moment().startOf('year').format('YYYY-MM-DD');
+const today = moment().subtract(1, 'day').format('YYYY-MM-DD');
+const firstDay = moment().subtract(6, 'months').format('YYYY-MM-DD');
+const lastDay = moment().add(6, 'months').format('YYYY-MM-DD');
+
+const BAR_WIDTH = 20;
```

Comment:
Can the bar width be narrower? It's nearly 1/2 the width of my text area, I think it could be reduced to 10 and still look great.